panel/src/components/Views/Users/ResetPasswordView.vue

```diff
@@ -27,20 +27,49 @@
  * @internal
  */
 export default {
+	props: {
+		/**
+		 * Enable the field for the current password of the acting user
+		 */
+		currentPassword: {
```

Comment:
`Suggestion:` How about naming this `requirePassword` instead? Distinguishes it more from `this.values.currentPassword`.